common/app/model/Tag.scala

```diff
@@ -286,4 +287,8 @@ case class Tag (
   val isFootballTeam = properties.references.exists(_.`type` == "pa-football-team")
   val isFootballCompetition = properties.references.exists(_.`type` == "pa-football-competition")
   val contributorImagePath = properties.bylineImageUrl.map(ImgSrc(_, Contributor))
+
+  override def branding(edition: Edition): Option[Branding] = {
```

Comment:
A capi `Tag` actually corresponds to a frontend `TagProperties`, which has an `activeBrandings` property: https://github.com/guardian/frontend/blob/master/common/app/model/Tag.scala#L240